### PR TITLE
Optimize extractSeriesFromGraph method in StatChecker

### PR DESCRIPTION
**Pull request by chejennifer**
- remove synchronized from extractSeriesFromGraph which is a method called for every node

### CODE REVIEW

File: util/src/main/java/org/datacommons/util/StatChecker.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
   // scaling factor, and unit of the stat var observations of the series summary.
   private final Map<String, SeriesSummary> seriesSummaryMap;
```
Comment (pradh) [inline util/src/main/java/org/datacommons/util/StatChecker.java:58]: Maybe its worth summarizing the concurrency handling of this class (spanning seriesSummaryMap, samplePlaces and svObsValues data structures)?  The stuff we discussed on chat...

**Review by pradh — APPROVED** [review]
Thanks for the change!

File: util/src/main/java/org/datacommons/util/StatChecker.java (hunk 1 above)
Comment (chejennifer) [reply util/src/main/java/org/datacommons/util/StatChecker.java:63]: done

**Comment by chejennifer** [comment]
Thanks for the review!

**Pull request merged**